cmd/spire-agent/cli/run/run.go

```diff
@@ -53,6 +53,10 @@ type agentConfig struct {
 
 	ConfigPath string
 	Umask      string `hcl:"umask"`
+
+	ProfilingEnabled string `hcl:"profiling_enabled"`
```

Comment:
Perhaps we can assume that profiling is enabled if ProfilingPort is set? And/or configure a default port and frequency, so we don't have to set three config vars every time?

Reply:
Perhaps. ProfilingEnabled was intended as a way to enable/disable profiling without loosing the port  and frequency settings. Port and Frequency serve different purposes and they don't depend on each other, you can set one and omit the other, the former sets the port where the pprof server listens, and the later is the frequency by which the profile data is taken by our profiling package. Then you can activate only the pprof server, or only our profiling component, or both. Anyway, I could remove ProfilingEnabled if you think it is easier to set/unset port or frequency to activate/deactivate each functionality. Just let me know and I will change it.